webapp/src/components/LinkedChannelCard/LinkedChannelCard.component.tsx

```diff
@@ -1,60 +1,128 @@
-                    text={msTeamsTeamName}
-                >
-                    <h5 className='my-0 opacity-6 msteams-linked-channel__body-values'>{msTeamsTeamName}</h5>
-                </Tooltip>
+            <div className='d-flex flex-column gap-6 msteams-linked-channel__body'>
+                <div className='d-flex gap-8 align-items-center'>
+                    {mattermostChannelType === MMConstants.PRIVATE_CHANNEL ? <Icon iconName='lock'/> : <Icon iconName='globe'/>}
+                    <Tooltip
+                        placement='left'
+                        text={mattermostChannelName}
+                    >
+                        <h5 className='my-0 msteams-linked-channel__body-values'>{mattermostChannelName}</h5>
+                    </Tooltip>
+                    <Tooltip
+                        placement='left'
+                        text={mattermostTeamName}
+                    >
+                        <h5 className='my-0 opacity-6 msteams-linked-channel__body-values'>{mattermostTeamName}</h5>
+                    </Tooltip>
+                </div>
+                <div className='d-flex gap-8 align-items-center'>
+                    <Icon iconName='msTeams'/>
+                    <Tooltip
+                        placement='left'
+                        text={msTeamsChannelName}
+                    >
+                        <h5 className='my-0 msteams-linked-channel__body-values'>{msTeamsChannelName}</h5>
+                    </Tooltip>
+                    <Tooltip
+                        placement='left'
+                        text={msTeamsTeamName}
+                    >
+                        <h5 className='my-0 opacity-6 msteams-linked-channel__body-values'>{msTeamsTeamName}</h5>
+                    </Tooltip>
+                </div>
             </div>
+            <Button
+                variant='text'
+                onClick={() => {
+                    setUnlinkChannelParams({channelId: mattermostChannelID});
+                    setShowUnlinkDialog(true);
+                }
+                }
+                className='msteams-linked-channel__unlink-icon'
+            >
+                <UILibIcon
+                    name='Unlink'
+                    size={16}
+                />
+            </Button>
+            <Dialog
+                show={showUnlinkDialog}
+                title='Unlink channels'
+                destructive={true}
+                primaryButtonText='Unlink channels'
+                secondaryButtonText='Cancel'
+                onSubmitHandler={unlinkChannel}
+                onCloseHandler={() => setShowUnlinkDialog(false)}
+            >
+                {isLoading && <LinearProgress className='absolute w-full left-0 top-62'/>}
+                <span>{'Are you sure you want to unlink the '}<b>{mattermostChannelName}</b>{' and '} <b>{msTeamsChannelName}</b>{' channels? Messages will no longer by synced.'}</span>
+            </Dialog>
+            <Dialog
```

Comment:
![Screenshot from 2023-12-14 09-43-20](https://github.com/Brightscout/mattermost-plugin-msteams-sync/assets/115625377/4d878d91-5ef9-48a1-a3a5-f4649be75505)
UI is breaking. Please Check

Reply:
Also we can avoid destructive dialog for Retry actions